PendingReleaseNotes

```diff
@@ -149,6 +149,17 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   and it can be disabled by using:
    `ceph config set mgr mgr/volumes/snapshot_clone_no_wait false`
 
+* CephFS: It is now not possible to fail an MDS without confirmation flag
+  when it has MDS_HEALTH_TRIM or MDS_HEALTH_CACHE_OVERSIZED health warnings.
+  This is to prevent accidental MDS failure with such health warnings because
+  restarting it may create new problems due to slow recovery.
+
+* CephFS: It is now not possible to fail an FS without confirmation flag
```

Comment:
reword: `fs fail` requires a confirmation flag when some MDSs exhibit certain health warning (<warning>,...). This is to prevent accidental MDS failover causing delays in recovery.

(and the above paragraph too)

Reply:
Done.